lib/core_ext/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -141,6 +145,29 @@ def find_column_names(table_name, index)
         column_names
       end
 
+      # Find not default operator class names for columns from index.
+      #
+      # @param column_names [Array] List of columns from index.
+      # @param index [Hash] index index attributes
+      # @return [Hash]
+      def find_operator_names(column_names, index)
+        column_names.each_with_index.inject({}) do |class_names, (column_name, column_index)|
```

Comment:
On the one hand, this is mildly concerning from an efficiency standpoint. On the other hand, I don't think it makes any tangible difference in real life. @baburdick, do you have any strong opinions on this?

Reply:
The main issue here is that column with list of operations is array of id. Same is for index columns. If this is critical I can try to rewrite this to use one query instead of separate for each column.

Reply:
Unless I'm mistaken, this occurs once at startup time, correct? If so, unless there's a circumstance in which this could impede clients of this gem, optimizing this can wait.

Reply:
schema.rb doesn't work in production. Usually it is used in test environment for creation database when we run rake db:test:prepare to create database structure.

Reply:
This should never run in production. Maybe theoretically in console, but that's a stretch.

Reply:
I ment schema.rb is not used to build database structure in production. Only one case when this code can be run is when we run `rake db:migration`. In this case rake task could recreate schema.rb and run this code.